packages/frontend-v2/src/components/pages/Payment/Payment.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import { IMembership } from '../../../libs/interfaces/membership.interface';
+import {} from '../../../services/member.service';
+import { membershipService } from '../../../services/membership.service';
+import { PaymentCard } from '../../molecules/paymentCard';
+
+export const Payment = () => {
+  const [memberships, setMemberships] = useState<IMembership[]>([]);
+
+  const getNotPaidMemberships = async (): Promise<void> => {
+    const memberships = await membershipService.getNotPaid();
+    setMemberships(memberships?.data);
+  };
+  useEffect(() => {
+    getNotPaidMemberships();
+  }, []);
+  const paymentReceived = async () => {
+    await getNotPaidMemberships();
+  };
```

Comment:
If you merge master on your current, you will have access to the hook "useGetAllFromService" that I have written and it can really simplify the code here.